### ISSUE TITLE
Slightly boost mutation chance when researched in the Escritoire

### ISSUE THREAD

**Issue by mezz** [ForestryMC/ForestryMC#697]
I just assumed that the Escritoire did this already. 
Seems like a nice feature to have for people going through bees the hard way.
Also add a config to adjust how much the boost is. 

I'm thinking of boosting the chance by half, with a maximum of a 5% boost.
That brings common bees from 15% to 20%, Diligent bees from 10% to 15%, and Heroic bees from 6% to 9%.

**Comment by MysteriousAges** [comment]
100% support this. The Escritoire is probably very underused.

**Comment by Vexatos** [comment]
Not just "the hard way". Remember this is the only way to find out about secret species' existence.

**Comment by SirSengir** [comment]
The escritoire is somewhat related to a few ideas I had kicking around in my head for a while when I developed Forestry, but which never came to anything:

Bees are fun, but walking all the mutation chains every time you start a new (SSP) world will grow tedious fairly quickly IMO. I suspect that very, very, very few people - even if interested in bees - will ever actually reach anything like near-completion of their collections, simply because most worlds will be discarded before. An initial idea of mine was to allow 'cloning' of bees which were researched in the escritoire and then bred. (Probably a bit of a simplistic mechanic though.)

Also somewhat related is an idea I implemented in my scraped bee-thing: breeding stocks. If you have a queen with matching alleles on all chromosomes, you can name her 'stock', which is then shown in tooltips. This way you'd always know whether the given bee is a Meadows with Fast production or a Imperial Bee with Short lifespan, etc. I also added a bunch of flavour mechanics to breeding stocks, i.e. statistics on how many bees have lived, what the latest generation is, a coat of arms, etc.

To give an idea what I am talking about, two screenshots I made back then: https://imgur.com/XfRWuPX,p56UEYA

Tl;dr: I agree that the escritoire gives too little incentives to actually use it at the moment.

**Comment by SandGrainOne** [comment]
Anecdotal comment:
I recently loaded a world from 1.6.4 into a 1.7.10 environment and I had all my old bees with their existing traits. I continued refinment of the production bees like if I left the world yesterday. Really cool.

I had lost all my breeding knowledge though, so I've been using the escritoire a bit to recreate my knowledge.

**Comment by mezz** [comment]
That looks pretty awesome and gives me some ideas. 
I'm only adding small things for now though, still in maintenance mode.
